packages/client/src/components/SafeSettings/components/Assets/AddCollection.js

```diff
@@ -30,36 +31,42 @@ const AddCollection = ({ onCancel, onNext }) => {
 
   return (
     <>
-      <div className="p-5">
-        <h2 className="is-size-4 has-text-black">Add Collection</h2>
+      <div className="px-5 pt-5 columns is-vcentered is-multiline is-mobile">
+        <h2 className="is-size-4 has-text-black column">Add NFT Collection</h2>
+        <span className="pointer" onClick={onCancel}>
+          <Close className="mr-4" />
+        </span>
```

Comment:
The close button and header title is now handled by Modal.js as a prop passed in.

Reply:
Moved for all the modals in this PR, there are some modals left that need to be resolved, there is a bug ticket for it https://linear.app/dappercollectives/issue/VSL-190